zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
For partner-built integrations, we recommend that integration dashboards also include your logo as well as a description of the data being presented.

Reply:
We'll adda logo and desc.